test/unit/ui/camera.test.js

```diff
@@ -1011,6 +1011,46 @@ test('camera', (t) => {
             camera.flyTo({ center: [170, 0], duration: 10 });
         });
 
+        t.test('does not pan eastward across the antimeridian if no world copies', (t) => {
+            const camera = createCamera();
+            camera.transform._renderWorldCopies = false;
```

Comment:
Please adjust the test so that it sets this option via the public API (an argument to the `Transform` constructor) rather than a private property.